.gitmodules

```diff
@@ -3,7 +3,8 @@
 	url = https://github.com/NOAA-EMC/CICE5
 [submodule "FMS"]
 	path = FMS
-	url = https://github.com/NOAA-EMC/FMS
+	url = https://github.com/NOAA-GFDL/FMS
+        branch = master
```

Comment:
Please check out tag 2019.01 revision: 708b8d5, which is used in s2s. We are working on bringing in the latest GFDL dycore with FMS release tag 2020.02 in ufs-weather-model. GFDL did test ufs-weather-model for FMS release, we do not directly use the top of master in ufs applications.

Reply:
When I checkout this branch, I do see tag 2019.01 and 708b8d5 as the hash. 

The branch=master also is present in the ufs-s2s-model .gitmodules.

Reply:
The FMS version is correct.